docs/sources/flow/reference/components/otelcol.exporter.otlp.md

```diff
@@ -0,0 +1,199 @@
+---
+aliases:
+- /docs/agent/latest/flow/reference/components/otelcol.exporter.otlp
+title: otelcol.exporter.otlp
+---
+
+# otelcol.exporter.otlp
+
+`otelcol.exporter.otlp` accepts telemetry data from other `otelcol` components
+and writes them over the network using the OTLP gRPC protocol.
+
+> **NOTE**: `otelcol.exporter.otlp` is a wrapper over the upstream
+> OpenTelemetry Collector `otlp` exporter. Bug reports or feature requests may
+> be redirected to the upstream repository.
+
+Multiple `otelcol.exporter.otlp` components can be specified by giving them
+different labels.
+
+## Usage
+
+```river
+otelcol.exporter.otlp "LABEL" {
+  client {
+    endpoint = "HOST:PORT"
+  }
+}
+```
+
+## Arguments
+
+`otelcol.exporter.otlp` supports the following arguments:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`timeout` | `duration` | Time to wait before marking a request as failed. | `"5s"` | no
+
+## Blocks
+
+The following blocks are supported inside the definition of
+`otelcol.exporter.otlp`:
+
+Hierarchy | Block | Description | Required
+--------- | ----- | ----------- | --------
+client | [client][] | Configures the gRPC server to send telemetry data to. | **yes**
```

Comment:
Why is "yes" in bold?

Reply:
Just to call attention to something being required; we do it for other component reference pages. 

Do you think we should change it to an unbolded yes/no consistently?

Reply:
Yes, sorry -- I didn't catch this in others.  I don't think bold is necessary.

Reply:
Ack, I'll remove the bold from here and open a separate PR to fix the existing content.